lib/atlas/dataset/derived_dataset.rb

```diff
@@ -0,0 +1,27 @@
+module Atlas
+  class Dataset::DerivedDataset < Dataset
+    GRAPH     = 'graph.yml'.freeze
+
+    attribute :base_dataset,      String
+    attribute :scaling,           Hash
```

Comment:
Instead of a Hash, what about reusing [Preset::Scaling](https://github.com/quintel/atlas/blob/master/lib/atlas/preset/scaling.rb)?